File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        // 10개면,
        // 10번 + 10번
        int odd = 0;
        int even = 0;
        for (int i = 0; i < num_list.length; i += 2) {
            odd += num_list[i];
        }
        for (int i = 1; i < num_list.length; i += 2) {
            even += num_list[i];
        }
        return odd > even ? odd : even;
    }
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    sum1 = sum(num_list[i] for i in range(0, len(num_list), 2))
    sum2 = sum(num_list[i] for i in range(1, len(num_list), 2))
    
    return max(sum1, sum2)